src/database.py

```
@@ -0,0 +1,14 @@
+from sqlalchemy.ext.asyncio import create_async_engine, async_sessionmaker
+from sqlalchemy.orm import declarative_base
+
+from src.settings import settings
+
+SQLALCHEMY_DATABASE_URL = settings.SQLITE_DATABASE_URL
```

`settings.SQLITE_DATABASE_URL` is typed as optional in settings; consider asserting / validating it's present here (or raising a helpful error) so a missing config fails early instead of producing confusing engine errors at runtime.